src/index.ts

```diff
@@ -53,20 +35,35 @@ const resolvers = {
   },
 
   Mutation: {
-    createUser: async (_, { data }) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    createUser: async (parent, args, context, info) => {
+      const { data } = args;
+      const connection: Connection = context.connection;
+
       const user = new User();
-      user.id = 20;
       user.name = data.name;
       user.email = data.email;
       user.password = data.password;
       user.birthDate = data.birthDate;
+
+      await connection.manager.save(user);
+
       return user;
     },
   },
 };
 
-const server = new ApolloServer({ typeDefs, resolvers });
+async function startServer() {
+  const connection = await createConnection().catch((error) => console.log(error));
+  const server = new ApolloServer({
+    typeDefs,
+    resolvers,
+    context: { connection: connection },
```

Comment:
```suggestion
    context: { connection },
```

quando a variável tem o mesmo valor da chave, pode omitir a variavel